examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    print(optime.ccode)
</pre><p>Possibly comment this and let the user uncomment if they want, as the C code generated is fairly long and breaks up flow of notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I would agree with that.</p>

Reply:
<p>I agree as well.</p><p>Something along the lines of</p><pre class="ql-syntax" spellcheck="false"># If you want to have a look at the generated code....uncomment...
#print(op.ccode)
</pre>

Reply:
<p>I do agree, makes sense, thx for the review!</p>